### PR TITLE
feat(pci): add cross border payment sentence for TN/SA/MN

### PR DESCRIPTION
**Pull request by darsene**
ref: MANAGER-11389

| Question         | Answer
| ---------------- | ---
| Branch?          | `master`
| Bug fix?         | no
| New feature?     | yes
| Breaking change? | no
| Tickets          | Fix #MANAGER-11389
| License          | BSD 3-Clause

- [x] Try to keep pull requests small so they can be easily reviewed.
- [x] Commits are signed-off
- [x] Only FR translations have been updated
- [x] Branch is up-to-date with target branch
- [x] Lint has passed locally
- [x] Standalone app was ran and tested locally
- [x] Ticket reference is mentioned in linked commits (internal only)
- ~Breaking change is mentioned in relevant commits~

## Related
:flags: Translation

- [x] TRANS-52464

### CODE REVIEW

File: packages/components/ng-ovh-payment-method/src/components/register/constants.js [hunk 1]
```diff
@@ -4,22 +4,22 @@ import BANK_ACCOUNT from './assets/bank_account.png';
...
+export const DEFAULT_SELECTED_PAYMENT_METHOD_TYPE = 'CREDIT_CARD';
+
+export const PAYPAL_PAYMENT_METHOD = 'PAYPAL';
+
 export const DEFAULT_ORDERED_PAYMENT_METHOD_TYPES = [
-  'CREDIT_CARD',
-  'PAYPAL',
+  DEFAULT_SELECTED_PAYMENT_METHOD_TYPE,
+  PAYPAL_PAYMENT_METHOD,
   'BANK_ACCOUNT',
```
Comment (sachinrameshn) [inline packages/components/ng-ovh-payment-method/src/components/register/constants.js:14]: You can create another constant for 'BANK_ACCOUNT' ? Is that a another type ?

Comment (darsene) [reply packages/components/ng-ovh-payment-method/src/components/register/constants.js:14]: Sure. I my case, I don't need to use 'BANK_ACCOUNT' but some constants values were duplicated that's why I refactor this part

File: packages/manager/tools/webpack-dev-server/src/config.js
Comment (darsene) [inline packages/manager/tools/webpack-dev-server/src/config.js:1]: `yarn start --env.local2API ` doesn't work anymore that's why I added  `yn(process.env.npm_config_local2API)`

To enable or disable : `npm config set local2API=[false|true]`

File: packages/components/ng-ovh-payment-method/src/components/register/constants.js (hunk 1 above)
Comment (darsene) [reply packages/components/ng-ovh-payment-method/src/components/register/constants.js]: @sachinrameshn It's fixed !

**Review by sachinrameshn — APPROVED** [review]

File: packages/components/ng-ovh-payment-method/src/components/register/index.html [hunk 2]
```diff
@@ -67,6 +67,10 @@
                 data-translate="ovh_payment_method_register_set_as_default_choice_information"
             ></span>
         </oui-message>
+        <p
+            data-ng-if="$ctrl.model.selectedPaymentMethodType.type.paymentType === $ctrl.CREDIT_CARD_PAYMENT_METHOD && $ctrl.hasSpecificCrossBorderSentenceForCardPayment"
```
Comment (sachinrameshn) [inline packages/components/ng-ovh-payment-method/src/components/register/index.html:71]: Suggestion: Can you move the check to controller for readability ?

Comment (darsene) [reply packages/components/ng-ovh-payment-method/src/components/register/index.html]: @sachinrameshn : fixed !

File: packages/manager/tools/webpack-dev-server/src/config.js [hunk 3]
```diff
@@ -21,7 +21,11 @@ module.exports = (env) => {
 
   const sso = new Sso(region, envConfig);
 
-  if (yn(env.local2API) || yn(process.env.npm_package_config_local2API)) {
+  if (
```
Comment (sachinrameshn) [inline packages/manager/tools/webpack-dev-server/src/config.js:24]: Question: Why are we adding npm_config_local2API here ?

Comment (darsene) [reply packages/manager/tools/webpack-dev-server/src/config.js:24]: see my last comment. In this way, I can activate local2API in the user .npmrc file

Comment (darsene) [reply packages/manager/tools/webpack-dev-server/src/config.js]: I removed the file form this commit @sachinrameshn

File: packages/components/ng-ovh-payment-method/src/components/register/controller.js [hunk 4]
```diff
@@ -82,104 +91,117 @@ export default class OvhPaymentMethodRegisterCtrl {
...
+              if (!get(paymentType, 'icon.data')) {
+                set(
+                  paymentType,
+                  'icon.data',
+                  get(FALLBACK_IMAGES, paymentType.paymentType),
+                );
+              }
+              return paymentType;
+            },
+          );
```
Comment (ghyenne) [inline packages/components/ng-ovh-payment-method/src/components/register/controller.js:135]:
```suggestion
          this.availablePaymentMethodTypes.list = this.availablePaymentMethodTypes.list.map(
            (paymentTypeParam) => {
              const icon = paymentTypeParam.icon?.data;
              return { ...paymentTypeParam, icon: { data: icon || FALLBACK_IMAGES[paymentType] }};
            },
          );
```

Comment (darsene) [reply packages/components/ng-ovh-payment-method/src/components/register/controller.js:135]: @ghyenne : this part of code was already existing, it doesn't concern my changes. I can modify if you want but your suggestion is not really optimized. Indeed, each object will be duplicated even if icon property is already set. Are you sure about that ?

**Review by ghyenne — APPROVED** [review]

**Review by sachinrameshn — DISMISSED** [review]

**Review by ghyenne — DISMISSED** [review]

**Review by sachinrameshn — DISMISSED** [review]

**Review by qpavy — APPROVED** [review]

**Review by JacquesLarique — APPROVED** [review]

**Pull request merged**